core/src/main/resources/lib/layout/copyButton/copyButton.js

```diff
@@ -4,7 +4,9 @@ Behaviour.specify(
   0,
   function (copyButton) {
     copyButton.addEventListener("click", () => {
-      if (isSecureContext) {
+      // HTMLUnit 2.70.0 does not recognize isSecureContext
+      // https://issues.jenkins.io/browse/JENKINS-70895
+      if (!window.isRunAsTest && isSecureContext) {
```

Comment:
Wouldn't

```suggestion
      if (!window.isSecureContext) {
```

be the simpler change (found in @jtnord's Jira comment)?

Reply:
I see documentation on MDN for the [global `isSecureContext`](https://developer.mozilla.org/en-US/docs/Web/API/isSecureContext).  In the MDN documentation for [`window`](https://developer.mozilla.org/en-US/docs/Web/API/Window), the `window.isSecureContext` heading links to the documentation of the global property.  I assumed that means the MDN recommends use of the global rather than using `window.isSecureContext`.  

There is not a `window.isSecureContext` page on MDN (404 from https://developer.mozilla.org/en-US/docs/Web/API/Window/isSecureContext ), though there was at one point as indicated by a link from a [stackoverflow article](https://stackoverflow.com/questions/40679961/how-can-i-detect-if-chrome-thinks-the-current-page-is-a-secure-origin).

I also believe that I read a recommendation on some location that the global `isSecureContext` was preferred.  However, I can't find that reference now.

I also prefer the added check of `window.isRunAsTest` because it makes it clear that this workaround can be removed when we update to an HTMLUnit version that recognizes the global `isSecureContext`.

Reply:
Fine for me either way, just seemed simpler.